JuliaGPU/session2/Feyaz/q1.jl

```diff
@@ -0,0 +1,39 @@
+
+#I tried it twice, and built in sort function was around 67+ ms, 
+# and insertion sort was around 1 ms.
+using BenchmarkTools
+arr = rand(Int, 1048576)    #2^20=1048576
+len = lastindex(arr)
+
+function quicksort!(array)
```

Comment:
There is an error in your implementation of quicksort. The final array is not sorted.